projects/1.18/assets/irongenerators/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+  "item.moregenerators.netherstar_upgrade": "网络升级",
+  "block.moregenerators.iron_generator": "铁制发电机",
+  "block.irongenerators.refined_obsidian_generator": "强化黑曜石发电机",
+  "block.moregenerators.certus_quartz_generator": "赛特斯石英水晶发电机",
+  "item.moregenerators.emerald_upgrade": "Emerald升级",
+  "block.moregenerators.icon": "Icon",
+  "subtitles.generatorambient": "",
+  "block.irongenerators.obsidian_generator": "黑曜石发电机",
+  "item.moregenerators.generator_upgrade_obsidianto_netheride": "Generator Upgrade: �8Obsidian �rto �7Netherite",
```

```suggestion
  "item.moregenerators.generator_upgrade_obsidianto_netheride": "发电机升级：§8黑曜石§r->§7下界合金",
```
这里可以参考Storage Drawers和Functional Storage的升级译名
不一定要按这个来（）
但是小节号还是打回去比较好